aquarius/app/util.py

```diff
@@ -207,3 +211,11 @@ def get_sender_from_txid(web3, txid):
     if transaction is not None:
         return transaction["from"]
     return None
+
+
+def rename_metadata_keys(bucket):
+    for d in bucket:
+        d["name"] = d.pop("key")
+        d["count"] = d.pop("doc_count")
```

Comment:
note that `d.pop` will raise an exception if the key is not in the dict

Reply:
the keys will be there even if empty.